xrrtc.js

```diff
@@ -141,8 +146,12 @@ class XRChannelConnection extends EventTarget {
     this.dialogClient.setState(key, value);
   }
 
-  deleteState(key) {
-    this.dialogClient.deleteState(key);
+  getAllKeys() {
+    this.dialogClient.getAllKeys();
+  }
+
+  runCode(obj) {
```

Comment:
Another place that might need to be a Promise.

Reply:
hmm what about now?

Reply:
This is still not async?

Reply:
1f02ce4